src/NodeParser/FunctionNodeParser.ts

```diff
@@ -64,12 +64,34 @@ export function getNamedArguments(
             // If it's missing a questionToken but has an initializer we can consider the property as not required
             const required = node.parameters[index].questionToken ? false : !node.parameters[index].initializer;
 
-            return new ObjectProperty(node.parameters[index].name.getText(), parameterType, required);
+            return new ObjectProperty(getParameterName(node.parameters[index].name, index), parameterType, required);
         }),
         false,
     );
 }
 
+function getParameterName(node: ts.BindingName, index: number) {
+    if (node.parent) {
```

Comment:
```suggestion
    if (node.parent && node.pos !== -1) {
```

Programatically created nodes don't have text information.

Reply:
I don't get why the condition `node.pos !== -1` should be added. `node.parent` should not be sufficient?